tapdb/assets_store.go

```diff
@@ -3542,6 +3542,12 @@ func (a *AssetStore) QueryParcels(ctx context.Context,
 					"anchor tx: %w", err)
 			}
 
+			// Fill in the anchor transaction's output pkScripts.
+			for i := range outputs {
+				outputs[i].Anchor.PkScript =
+					anchorTx.TxOut[i].PkScript
```

Comment:
This is not correct, the indexes won't always match up.
This would be correct:
```
			for i, out := range outputs {
				outIdx := out.Anchor.OutPoint.Index
				pkScript := anchorTx.TxOut[outIdx].PkScript
				outputs[i].Anchor.PkScript = pkScript
			}
```

Reply:
Thanks, fixed!